### PR TITLE
Polish stats page

### PR DESCRIPTION
**Pull request by vkulinich-cl**

### CODE REVIEW

**Comment by green-jay** [comment]
some of the 4pool trades are actually USDT trades
https://hydradx.subscan.io/extrinsic/0x1a14e96587b6c720a629f419fd9bf2e4100d52c0dcaea10d48182ed09756754d
image
image

**Comment by vkulinich-cl** [comment]
> TVL is doubled, can be sourced from `https://api.hydradx.io/hydradx-ui/v1/stats/tvl` to be aligned image

this is because of the 0 id tvl bug on the BE

**Comment by green-jay** [comment]
> this is because of the 0 id tvl bug on the BE

@vkulinich-cl why does hdx TVL matter here? shouldn't we use the EP for total TVL i mentioned above?

File: src/sections/stats/StatsPage.tsx
Comment (jvonasek) [inline src/sections/stats/StatsPage.tsx:1]: remove border style at `subHeaderStyle`, it was used with old background, but now it should be the same as Bonds detail page

Comment (jvonasek) [reply src/sections/stats/StatsPage.tsx:1]: resolved

**Pull request merged**